evap/staff/templates/staff_questionnaire_index_list.html

```diff
@@ -32,7 +34,13 @@
                                     {% blocktranslate count questionnaire.contributions.count as count %}used {{ count }} time{% plural %}used {{ count }} times{% endblocktranslate %}
                                 </td>
                                 <td>
-                                    {% if type != 'contributor' %}
+                                    {% if type == 'dropout' %}
+                                        <!-- TODO@Felix: Selection via AJAX-->
+                                        <a href="{% url 'staff:questionnaire_set_active_dropout' questionnaire.id %}"
+                                           class="btn btn-sm btn-light {% if questionnaire.is_active_dropout_questionnaire %} active{% endif %}">
```

Comment:
We commonly do
```suggestion
                                           class="btn btn-sm btn-light{% if questionnaire.is_active_dropout_questionnaire %} active{% endif %}">
```
so that the resuling whitespace after template processing is canonical